src/Goerli-DssSpell.sol

```diff
@@ -16,26 +16,52 @@
 
 pragma solidity 0.6.12;
 // Enable ABIEncoderV2 when onboarding collateral through `DssExecLib.addNewCollateral()`
-pragma experimental ABIEncoderV2;
+// pragma experimental ABIEncoderV2;
 
 import "dss-exec-lib/DssExec.sol";
 import "dss-exec-lib/DssAction.sol";
 
 import { DssSpellCollateralAction } from "./Goerli-DssSpellCollateral.sol";
 
-interface StarknetBridgeLike {
-    function close() external;
-    function isOpen() external returns (uint256);
+interface TeleportJoinLike {
```

Comment:
Not mandatory but I really wish the order of as many things as possible in this file was aligned to the 30/8 spell were optimism and arbitrum teleports were onboarderd (for example interface functions in `TeleportBridgeLike`, or where "fees" are set with relating to the validation checks).
This would reduce the chances for anything being left out in this or future spells.

No idea why it isn't like that but basically it means doing the diff and shifting the relevant parts in this file.

Reply:
I'll see if I can painlessly refactor this.